test/src/endpoints/cart-endpoints.spec.js

```diff
@@ -31,15 +32,20 @@ afterEach(() => { nock.cleanAll() })
 
 describe('getCartV1', () => {
   test('fetches cart from the api', () => {
-    const cartId = '35'
+    const queryObject = {
+      include: 'line_items.item.product,line_items.line_item_discounts,discount_summaries,customer_account,billing_address,shipping_address,shipping_method'
```

Comment:
If we're specifying `include`, we should also specify the necessary `fields` for each include too.